### DISCUSSION TITLE
Login issue : Failed to update user's profile

### DISCUSSION THREAD

**Discussion by hongsw** [supabase/supabase#9487] [Questions]
Hi guys.

When I log in to `supabase.com` with my github ID, I get the following **"Failed to update user's profile"** error on `https://app.supabase.com/#` .

```
Request URL: https://api.supabase.io/platform/profile
Request Method: GET
Status Code: 409 
Response : {"message":"Failed to update user's profile","errorEventId":"e14bed5e74e7435b83ba5e3c57176c77"}
```

Can someone point me in the right direction? Thanks! 😄 

Note: 
I added a row last night and invited collaborators only. So it didn't cause any problems until last night.

**Comment by Sukikiroi** [comment]
Hello, 
 it's not the library's fault... seems that your dns provider is returning different ip addresses for api.supabase.com, some of which is unavailable.
I think  changing the DNS Can Help you

**Reply by hongsw** [reply]
Thanks Sukikiroi
```
$ dig api.supabase.com +short
supabase-api.fly.dev.
213.188.196.91
$ dig supabase-api.fly.dev +short
213.188.196.91
```
DNS is fine. 

I found a major cause, console following error.
1. UserObject missing.
```
UserObject missing! You should pass a UserObject to getValue(), in order to make targeting work properly. Read more: https://configcat.com/docs/advanced/user-object
```
<img width="512" alt="image" src="https://user-images.githubusercontent.com/1100974/195282566-fdbb1378-60cd-47ac-ad2c-a4e4a1a4912a.png">

2. So, Failed to update user's profile.
in webpack://_N_E/node_modules/@sentry/utils/esm/instrument.js
 

Note 1: I tried to log in with Chrome secret tab also, but fails too.
Note 2: My co-worker in same office is fine.
So, am i blocked?
